src/lib/assetBridger/ethBridger.ts

```diff
@@ -63,10 +61,15 @@ export interface EthWithdrawParams {
 
 export type EthDepositParams = {
   /**
-   * The L1 entity depositing the assets
+   * The L1 provider or signer
    */
   l1Signer: Signer
 
+  /**
+   * Address that is depositing the assets
+   */
+  from: string
```

Comment:
Why do we have this, it's not possible for the from address to be different from `l1Signer.getAddress()` is it?